WcaOnRails/app/controllers/translations_controller.rb

```diff
@@ -4,11 +4,18 @@ class TranslationsController < ApplicationController
   before_action :authenticate_user!, except: [:index]
 
   def self.bad_i18n_keys
-    @bad_keys ||= (I18n.available_locales - [:en]).each_with_object({}) do |locale, hash|
-      ref_english = Locale.new('en')
-      missing, unused, outdated = Locale.new(locale, true).compare_to(ref_english)
-      hash[locale] = { missing: missing, unused: unused, outdated: outdated }
-    end
+    @bad_keys ||= begin
+                    english = locale_to_translation('en')
```

Comment:
I noticed we were recomputing the English translation for each pass through the loop, which feels wasteful, so I pulled it out of the loop. I didn't do any benchmarking, but I imagine this will help performance a bit!